spring-cloud-generator/generate-one.sh

```diff
@@ -110,9 +110,6 @@ add_module_to_pom () {
     echo "adding module $starter_artifactid to pom"
     sed -i "/$2/a\ \ \ \ <module>"$starter_artifactid"</module>" $1
     if [[ ${3:-1} -eq 1 ]]; then
-      # we need to remove any previously existing entries so the entry points to the
-      # right $monorepo_commitish's readme
-      sed -i "/$starter_artifactid/d" README.md
```

Comment:
Removed since we are now using the [template](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/b256dc5e0bbdc9694ee3a2a19c2b5d91b30361ef/spring-cloud-generator/reset-previews-folder.sh#L7-L9) with an empty readme, per https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1529#discussion_r1088223388